docs/UserGuide.md

```diff
@@ -200,14 +201,17 @@ Examples:
 
 ### Finding patient details by nric: `find_nric`
 
-Finds patient who has the given nric.
+Finds the patient who has the given NRIC.
 
-Format: `find_nric NRIC`
+Format: `find_nric <NRIC>`
```

Comment:
could you remove the < >, so as to follow the format of other commands

Reply:
done !